packages/validator/templates/1.0.0/apiSpecifications.json

```diff
@@ -0,0 +1,186 @@
+{
+  "servers": {
+    "__maxSize": 1,
+    "__arrayItem": {
+      "url": {
+        "__regexp": "^(([^:\\/?#]+):)?(\\/\\/([^\\/?#]*))?([^?#]*)$"
```

Comment:
Was this copied from somewhere or did you write this?

Reply:
This should be copied from what was suggested in [previous PR](https://github.com/api3dao/airnode/pull/195#discussion_r578886984)